src/http/cookie.cr

```diff
@@ -56,7 +32,75 @@ module HTTP
         header << "; domain=#{domain}" if domain
         header << "; Secure" if @secure
         header << "; HttpOnly" if @http_only
+        header << "; #{@extension}" if @extension
+      end
+    end
+
+    module Parser
```

Comment:
This probably should be a `# :nodoc:` or the methods exposed here should at least point out that the primary interface is `Cookies.from_headers`